challenge/src/components/SomeInformationForTimeTravelers.jsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react'
+
+export default class SomeInformationForTimeTravelers extends React.Component {
+    render() {
+        var date = new Date();
+        var monthNum = date.getMonth();
+        var month;
+        // this is gonna be a lotta if statements rip
+        if(monthNum===1) {
```

Comment:
pls, consider some way of refactoring this

Reply:
i feel attacked but also yes i agree